src/main/kotlin/com/cognifide/gradle/aem/common/instance/tail/Log.kt

```diff
@@ -46,7 +47,7 @@ class Log(
 
     companion object {
 
-        private const val TIMESTAMP = """(?<timestamp>[0-9]{2}\.[0-9]{2}\.[0-9]{4}\s[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]{3})"""
+        private const val TIMESTAMP = """(?<timestamp>(^[\d[^(\*)]]+))"""
```

Comment:
is it still used?

Reply:
yep, it's used to pull out timestamp from log despite date format